packages/tradershub/src/components/RegulationSwitcher/RegulationSwitcherDesktop.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { useActiveTradingAccount, useAuthorize, useTradingAccountsList } from '@deriv/api';
+import { Button, qtJoin, Text } from '@deriv/quill-design';
+import { LabelPairedCircleInfoMdRegularIcon } from '@deriv/quill-icons';
+
+const RegulationSwitcherDesktop = () => {
+    const { switchAccount } = useAuthorize();
+    const { data: tradingAccountList } = useTradingAccountsList();
```

Comment:
```suggestion
    const { data: tradingAccountsList } = useTradingAccountsList();
```

accounts seem more reliable